docker-entrypoint.sh

```diff
@@ -68,16 +68,16 @@ else
         gsutil -m cp $NOMINATIM_GS_BUCKET/$NOMINATIM_DATA_LABEL/*.tgz* $NOMINATIM_DATA_PATH
 
         # Remove any files present in the target directory
-        rm -rf $NOMINATIM_POSTGRESQL_DATA_PATH/*
-        
+        rm -rf ${NOMINATIM_POSTGRESQL_DATA_PATH:?}/*
```

Comment:
Probably the one that will be the odd need to lookup so to save a bit of time from bash's man page:

```
      ${parameter:?word}
              Display Error if Null or Unset.  If parameter is null or  unset,
              the  expansion  of  word (or a message to that effect if word
              is  not present) is written to the standard error and the shell,
              if it is not interactive, exits.  Otherwise, the value of parameter
              is substituted.
```